.github/workflows/coverity.yml

```diff
@@ -105,6 +105,11 @@ jobs:
           mkdir -p ${RUNNER_TEMP}/${{ env.COVERITY }}
           tar zxf ${RUNNER_TEMP}/${{ env.COVERITY }}.tgz -C ${RUNNER_TEMP}/${{ env.COVERITY }} --strip-components 1
 
+      - name: Compile Coverity
```

Comment:
why no skip test exec?

Reply:
Accidentally missing, I thought I had added it to all. Fixed